packages/http-server/src/__tests__/body-params-validation.spec.ts

```diff
@@ -0,0 +1,258 @@
+import { createLogger } from '@stoplight/prism-core';
+import fastify = require('fastify');
+import { relative, resolve } from 'path';
+import { createServer } from '../';
+import { IPrismHttpServer } from '../types';
+
+const logger = createLogger('TEST', { enabled: false });
+
+async function instantiatePrism(fixture: string) {
+  const server = createServer({}, { components: { logger }, config: { mock: { dynamic: false } } });
+  await server.prism.load({
+    path: relative(process.cwd(), resolve(__dirname, 'fixtures', fixture)),
+  });
+  return server;
+}
+
+const expectPayload = (response: fastify.HTTPInjectResponse) => {
```

Comment:
I have some concerns here.

1. Payload is a generic term which often means request body but could mean request or response, but here it always means response.
2. Should this be an expect wrapper? could it be a custom matcher? 
3. this is always JSON only, we are meant to be JSON and XML at least, especially with #456 coming up this/next sprint